tests/eventdispatcher/PatientFilterEventHookTest/sql/install.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS `patient_filter_event_hook`;CREATE TABLE `patient_filter_event_hook` ( `id` INT NOT NULL AUTO_INCREMENT , `name` VARCHAR(255) NOT NULL , `route` VARCHAR(255) NOT NULL , `description` VARCHAR(255) NOT NULL , PRIMARY KEY (`id`)) ENGINE = InnoDB;INSERT INTO `patient_filter_event_hook` (`id`, `name`, `route`, `description`) VALUES (NULL, 'test_hook1', '/test_hook/', 'test hook'), (NULL, 'test_hook_detail', '/test_hook_detail/', 'test hook detail');
```

Comment:
break this into lines
also, does this table do anything?

Reply:
This is a test table, just for an example. Break this into lines is not possible. because in function installSQL using explode by ";". For back compatibility , i saved this code.